Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4600.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.ManualReview)]
+	[Explicit] // Marked explicit until we have a lane for CollectionView tests
```

Comment:
this should keep the test of running on AppCenter right? it isn't .. it's failing the test.

Reply:
It should have, but it seems UITests is doing something weird. Changed it to Ignore, we'll hope that works.